scripts/download_mmctl_release.sh

```diff
@@ -15,14 +15,8 @@ fi
 
 BIN_PATH=${2:-bin}
 
-# strip whitespace
-THIS_BRANCH=$(git rev-parse --abbrev-ref HEAD)
-if [[ "$THIS_BRANCH" =~ 'release-'[0-9] ]];
-then
-  RELEASE_TO_DOWNLOAD="$THIS_BRANCH"
-else
-  RELEASE_TO_DOWNLOAD=master
-fi
+RELEASE_TO_DOWNLOAD=release-6.2
```

Comment:
@puerco i was trying to understand the mmctl release and things and notice to pin on this is the best approach for now and to reduce the work from us